linode/manager
# Api cli close modal on linode create

## Description 📝
Address UX Feedback
Automatically closes commands modal upon creating Linode from terminal.

## How to test 🧪

- Go to Linode create page and fill required fields.
- Click on Create Command Line button.
- Copy command and execute in terminal.
- Observe behavior of closing modal and navigating user back to Linode's page.

Test creating Linode, clone Linode, Deploy Linode from backup

**How do I run relevant unit or e2e tests?**

- yarn test
- yarn cy:run
